static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -415,7 +415,6 @@ function OnboardingContent({
         projectSlug={currentProject.slug}
         configType={
           setupMode() === 'npm' || // switched to NPM option
-          (!setupMode() && defaultTab === 'npm') || // default value for FE frameworks when ?mode={...} in URL is not set yet
```

Comment:
note: I removed this condition because with the current code, it can never yield `true`. `defaultTab` was changed to unconditionally be `'jsLoader'` in [#79421](https://github.com/getsentry/sentry/pull/79421/files#diff-77e24a62b1a771eb1cee889b0e94664634c6a1cf97ff18c11b2fb2ab632df1a5L196-R196), so TypeScript now complains that comparing it with `'npm'` doesn't make sense.

@priscilawebdev FYI